arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -283,23 +253,29 @@
         ],
         "responses": {
           "200": {
-            "description": "",
+            "description": "OK -- List of storage accounts was retrieved and returned successfully.",
```

Comment:
I changed it for consistency with the latest swagger . I will leave it as such matching to the latest
https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-12-01/swagger/storage.json#L269